deepchem/models/tests/test_layers.py

```diff
@@ -1544,3 +1544,28 @@ def test_torch_graph_gather():
         np.load("deepchem/models/tests/assets/graphgatherlayer_result.npy"),
         atol=1e-4)
     assert result.shape == (batch_size, 2 * n_features)
+
+
+@pytest.mark.torch
+def test_torch_cosine_dist():
```

Comment:
Can you please add another test that uses pre calculated values for a small tensor and verify if the result is as expected?